build/main.go

```diff
@@ -107,7 +139,8 @@ func rubyBuild(ctx context.Context, client *dagger.Client, hostDirectory *dagger
 		WithWorkdir("/app").
 		WithExec([]string{"bundle", "install"}).
 		WithExec([]string{"bundle", "exec", "rake", "build"}).
-		Sync(ctx)
+		File("/app/pkg/flipt_client-0.1.0.gem").
```

Comment:
TODO: I tried using a wildcard here since the version will change.. but it seems Dagger does not expand wildcards?

```
┃ 2023/12/21 17:05:46 input:1: container.from.withDirectory.withDirectory.withWorkdir.withExec.withExec.file lstat /app/pkg/flipt_client*.gem: no such file or directory
┃ exit status 1
```